src/riakc_ts_query_operator.erl

```diff
@@ -31,9 +31,12 @@
 -export([serialize/2,
          deserialize/1, deserialize/2]).
 
-serialize(QueryText, Interpolations) ->
+
+serialize(QueryText, Interpolations)
+  when is_binary(QueryText) orelse is_list(QueryText) ->
+    Q = unicode:characters_to_binary(QueryText),
```

Comment:
Of course, I don't know why I thought `error` would magically get returned. I'll make the change.